src/producer.rs

```diff
@@ -431,13 +471,31 @@ pub fn producer(
             } else {
                 path_dir
             };
-            archives.push(Archive {
-                name: path.to_string(),
-                item: RefCell::new(ArchiveType::Dir(full_path)),
-            });
+            if full_path.is_dir() {
+                archives.push(Archive {
+                    name: path.to_string(),
+                    item: RefCell::new(ArchiveType::Dir(full_path)),
+                });
+            } else if let Some(ext) = full_path.clone().extension() {
+                let ext = ext.to_str().unwrap();
+                if ext == "info" || ext == "json" {
```

Comment:
We could handle xml files too

Reply:
we could have several jacoco files coming from different machines